qf_lib/brokers/binance_broker/binance_broker.py

```diff
@@ -64,14 +64,15 @@ class BinanceBroker(Broker):
 
```

Comment:
Maybe we can add the docstring  to the BinanceAccountSettings?

```
    account_name: Optional[str]
        Name of the account to which the broker will be connected.
        It is useful when more than one instance of Broker is running in order to differentiate transactions
```

Reply:
Added